hydroshare/settings.py

```diff
@@ -706,6 +706,7 @@
 TASK_NAME_MAPPING = {
     'hs_core.tasks.create_bag_by_irods': 'bag download',
     'hs_core.tasks.create_temp_zip': 'zip download',
+    'hs_core.tasks.unzip_task': 'unzip file',
```

Comment:
Should we standardize the task name convention a bit across the tasks we have so far by naming it ```file unzip``` or the other way around by naming other tasks as ```download bag```, etc.? A minor issue, but I think standardizing the naming convention is a good idea.

Reply:
agreed, thanks for pointing this out.  done.